portal/templates/_partials/primary-navigation.njk

```diff
@@ -32,7 +36,7 @@
   ]
 }) }} 
 
-{% elif user %}
+{% elif isChecker or isMaker %}
```

Comment:
Does the UI allow a maker to be an `admin` as well so a combined `maker-admin` role is possible in the latest design?